js/packages/teams-ai/src/MemoryFork.ts

```diff
@@ -73,7 +73,10 @@ export class MemoryFork implements Memory {
      */
     public deleteValue(path: string): void {
         const { scope, name } = this.getScopeAndName(path);
-        if (this._fork.hasOwnProperty(scope) && this._fork[scope].hasOwnProperty(name)) {
+        if (
+            Object.prototype.hasOwnProperty.call(this._fork, scope) &&
```

Comment:
just out of curiosity, why are you changing this syntax since theoretically it should be the sample outcome?

Reply:
The security linter that was installed on the project `eslint-plugin-security` requires `hasOwnProperty` to be used this way. I actually do wonder about how necessary this is, as well as a bunch of other linting errors that it brings up -- would be nice if we can discuss possibility of removing that plugin. If we do that, it would reduce our linting errors to less than 100.

Reply:
(Atm I think I have reduced the linting errors to be around 500)